packages/SymbioticController/src/Adapter/Nette/Application/PresenterFactory.php

```diff
@@ -62,8 +62,12 @@ public function createPresenter($name)
      */
     public function getPresenterClass(&$presenterName): string
     {
-        if (isset($this->cache[$presenterName])) {
-            return $this->cache[$presenterName];
+        if (class_exists($presenterName)) {
```

Comment:
This seems to be bug for me. If you pass `Error` to `$presenterName` it will return `TRUE` to `class_exists()` condition, but it is not presenter's class name. This is reason why it was written like this: https://github.com/Lexinek/SymbioticController/blob/master/src/Application/PresenterFactory.php#L72